### ISSUE TITLE
Using 'p' as an alias for 'on' event attribute

### ISSUE THREAD

**Issue by biphasic** [neuromorphicsystems/event_stream#2]
Tonic works with the event attributes [x, y, t, p] (in any order, as a structured numpy array), while event_stream uses the attribute 'on' instead of 'p'. What I then typically do, when loading events with event_stream and processing them in Tonic, is to add a [field title](https://numpy.org/doc/stable/user/basics.rec.html#field-titles), which acts as an alias for the 'on' attribute. 
```
events = events_from_event_stream.view(dtype=events.dtype.descr[:3] + [(('on', 'p'), '?')])
assert all(events['p'] == events['on'])
```
Would it be possible to pull this upstream into the event_stream library?

**Comment by aMarcireau** [comment]
@biphasic I'll take a look and see if I can add it. The dtype is defined in the C extension so I need to figure out how to add a field title with the C API. I might to that whilst figuring out what needs updating for numpy 2.0.

**Comment by aMarcireau** [comment]
@biphasic Implemented in version 1.6.1.
